### PR TITLE
Auto assign reviewers

### PR DESCRIPTION
**Pull request by erwinmombay**
we will auto assign reviewers only on newly opened pull requests

### CODE REVIEW

**Comment by erwinmombay** [comment]
@rsimha PTAL

File: routes/owners/api.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export function index(req, res) {
     return body && body.pull_request && prActionTypes.indexOf(body.action) > -1;
   }
 
-  function isReviewAction() {
+  function isReviewAction(body) {
```
Comment (rsimha) [inline routes/owners/api.js]: Curious: Was this function using a different `body` earlier? Or is this a bug that got fixed?

File: tests/test-api.js [hunk 2]
```diff
@@ -50,6 +50,22 @@ test.serial('on an opened pull request, it should set initial reviewers', t => {
       });
 });
 
+test.serial('on an sync pull request, it should not set reviewers', t => {
```
Comment (rsimha) [inline tests/test-api.js]: nit: `on a sync...`

File: routes/owners/api.js (hunk 1 above)
Comment (erwinmombay) [reply routes/owners/api.js]: because of js closures, its the same `body` from the outer scope, i just wanted to be explicit about passing it in.

**Pull request merged**